src/main/kotlin/net/axay/kspigot/items/KSpigotItems.kt

```diff
@@ -104,9 +113,9 @@ fun ItemMeta.removeFlags(vararg itemFlag: ItemFlag) = removeItemFlags(*itemFlag)
 /**
  * Provides safe access to the items' displayName.
  */
-var ItemMeta.name: String?
-    get() = if (hasDisplayName()) displayName else null
-    set(value) = setDisplayName(if (value == null || value == "") " " else value)
+var ItemMeta.name: Component?
```

Comment:
could you introduce a new `var` called `stringName`, but deprecate it instantly?